projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/README.md

```diff
@@ -50,6 +50,12 @@ the frontend extension, check the frontend extension is installed:
 jupyter labextension list
 ```
 
+If you are struggling with a particular aspect of the JupyterLab API,
+you can contact the Jupyter team in the following way: go to their repo
+(https://github.com/jupyterlab/jupyterlab), go to Issues, go to New issue,
```

Comment:
nit: instead of instructions just link https://github.com/jupyterlab/jupyterlab/issues/new/choose ?